handwriting_recognition.ipynb

```diff
@@ -0,0 +1,1027 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    def prepare_image_paths(samples):
</pre><p>Can we rename this to something like <code>get_image_path_and_labels(..)</code>?  Any name that we use in the code should be self-explanatory as much as possible</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sayakpaul/Handwriting-Recognizer-in-Keras/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>